contributing.md

```diff
@@ -1,3 +1,27 @@
+# Contributing
+
+Contributions to go-nitro are welcome from anyone.
+
+Magmo (at Consensys Mesh) are developing go-nitro under contract for the Filecoin Foundation. As a consequence, we are obligated to maintain a certain level of quality.
+
+## Contribution lifecycle
+
+Magmo uses [Github Projects](https://github.com/statechannels/go-nitro/projects/1) for go-nitro project management. We will aim to keep a handful of "good first issues" in [the pipeline](https://github.com/statechannels/go-nitro/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), which are a good place to start.
+
+Before spending lots of time working on an issue, consider asking us for feedback. We would love to help make your contributions more successful!
```

Comment:
Consider:
```suggestion
Before spending lots of time working on an issue, consider asking us for feedback via [Slack](https://statechannels.slack.com/archives/C02J81JFD3J), the [issue tracker](https://github.com/statechannels/go-nitro/issues), or [email](magmo@mesh.xyz). We would love to help make your contributions more successful!
```

caveat: I'm not sure whether that email link will render as a `mailto`.